### DISCUSSION TITLE
Will pnpm clean global store automatically？

### DISCUSSION THREAD

**Discussion by smallnine9** [pnpm/pnpm#5655] [Q&A] [answered]
If I have many projects using pnpm, there are many dependency packages in my global store.
If I remove these projects, would theses dependency pakcages in global store would be deleted automatically?

**Comment by lvqq** [comment, marked as answer]
The dependency packages won't be removed automatically. You can run `pnpm store prune` to remove them manually. https://pnpm.io/cli/store

**Reply by smallnine9** [reply]
get it, thank you!
